ui/package-lock.json

```diff
@@ -1,17213 +1,8 @@
 {
   "name": "questdb-console",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
why this change is here?

Reply:
i have no idea, this file has a life of its own - i ran `mvn package` and here we are